nose2pytest/script.py

```diff
@@ -528,17 +528,33 @@ def _transform_dest(self, assert_arg_test_node: PyNode, results: {str: PyNode})
         adjust_prefix_first_arg(new_aaa, results["aaa"].prefix)
 
         dest2 = self._get_node(assert_arg_test_node, self._arg_paths[1])
-        dest2.replace(wrap_parens_for_addsub(bbb))
+        new_bbb = wrap_parens_for_addsub(bbb)
+        if get_prev_sibling(dest2).type in NEWLINE_OK_TOKENS:
+            new_bbb.prefix = ''
+        dest2.replace(new_bbb)
 
         dest3 = self._get_node(assert_arg_test_node, self._arg_paths[2])
         if delta.children[0] == PyLeaf(token.NAME, 'delta'):
             delta_val = delta.children[2]
-            delta_val.prefix = " "
-            dest3.replace(wrap_parens_for_comparison(delta_val))
+            delta_val.prefix = ""
+            wrapped_delta_val = wrap_parens_for_comparison(delta_val)
+            dest3.replace(wrapped_delta_val)
+
+        elif delta.children[0] == PyLeaf(token.NAME, 'places'):
```

Comment:
I've never modified the `PATTERN_ALMOST_ARG`.
https://github.com/pytest-dev/nose2pytest/blob/304d8755647b2b38cc9c0e51fba5691472beff21/nose2pytest/script.py#L74-L78